<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection</class>
 <widget class="QWidget" name="Connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:transparent;</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(86, 86, 86, 86, 0.7);</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>563</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="connectionLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#connectionLabel {
	background:rgba(3, 151, 229, 0.7);
	font: 75 14pt &quot;MS Shell Dlg 2&quot;;
	color: white;
	font-weight: bold;
	margin-bottom: 10px;
}</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QLineEdit" name="usernameEdit">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#usernameEdit {
	border-radius: 10px;
	background: url(C:/Everywhere/images/account.png)  no-repeat;
	padding-left: 45 px;
	color: rgb(3, 151, 229);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 1px solid rgb(3, 151, 229);
}</string>
       </property>
       <property name="text">
        <string>Username or email</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="passwordEdit">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit#passwordEdit {
	border-radius: 10px;
	background: url(C:/Everywhere/images/lock.png)  no-repeat;
	padding-left: 45 px;
	color: rgb(3, 151, 229);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 1px solid rgb(3, 151, 229);
}</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="connectButton">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#connectButton {
	background: url(C:/Everywhere/images/backgroundbutton.png) no-repeat;
	border-radius:10px;
	color: white;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	font-weight: bold;
}

QPushButton#connectButton:pressed {
	background: url(C:/Everywhere/images/backgroundbutton-pressed.png) no-repeat;
}</string>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="forgotPassword">
       <property name="styleSheet">
        <string notr="true">QPushButton#forgotPassword {
	border:none;
	color: white;
	
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: underline;
	background:transparent;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>Forgot password ?</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="googleButton">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#googleButton {
	background: url(C:/Everywhere/images/google-signup.png) no-repeat;
	border-radius:10px;
}

QPushButton#googleButton:pressed {
	background: url(C:/Everywhere/images/google-signup-pressed.png) no-repeat;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="facebookButton">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#facebookButton {
	background: url(C:/Everywhere/images/facebook-signup.png) no-repeat;
	border-radius:10px;
}

QPushButton#facebookButton:pressed {
	background: url(C:/Everywhere/images/facebook-signup-pressed.png) no-repeat;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="orLabel">
       <property name="styleSheet">
        <string notr="true">QLabel#orLabel {
	border:none;
	color: white;
	
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: underline;
	background:transparent;
	font-weight: bold;
}</string>
       </property>
       <property name="text">
        <string>No account ?</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="signinButton">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>190</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#signinButton {
	background: url(C:/Everywhere/images/backgroundbutton.png) no-repeat;
	border-radius:10px;
	color: white;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	font-weight: bold;
}

QPushButton#signinButton:pressed {
	background: url(C:/Everywhere/images/backgroundbutton-pressed.png) no-repeat;
}</string>
       </property>
       <property name="text">
        <string>SIGN IN</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="ring">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="warning">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
